documentation/modules/rql/pages/user-guide.adoc

```diff
@@ -114,59 +120,73 @@ option=limit(_start_,_count_),sort(-+attribute1,attribute2,attributeN)
 
 === Sorting
 
-Sorts the data based on the specified attribute in the order indicated by the prefix (+ for ascending, - for descending) The attributes names are separated by a comma.
+Sorts the data based on the specified attribute in the order indicated by the prefix (+ for ascending, - for descending). The attribute names are separated by a comma.
+
 With user-defined sorting, the client is responsible for preventing an arbitrary order of the retrieved results.
 
 [subs="quotes"]
 ----
 option=sort(-+attribute1,attribute2,attributeN)
 ----
+//TODO ? Why -+? Wouldn'T it suffice to only add + and the attribute names? Is it required to always note down - and +? Does the order matter?
```

Comment:
option=sort(-attribute1,+attribute2,[+-]attributeN)

Yes order is important

Reply:
Done -- https://github.com/bci-oss/semanticstack-rql/pull/69/commits/2192784ca4ef47774b167c992259c2fc5b2632d5